Tests/SentryTests/Performance/SentryTracerTests.swift

```diff
@@ -266,6 +298,87 @@ class SentryTracerTests: XCTestCase {
         XCTAssertEqual(1, transactionsWithAppStartMeasrurement.count)
         XCTAssertNil(SentrySDK.getAndResetAppStartMeasurement())
     }
+
+    #if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+
+    func testChangeStartTimeStamp_RemovesFramesMeasurement() {
+        let sut = fixture.getSut()
+        givenFrames(1, 1, 1)
+        sut.startTimestamp = Date()
```

Comment:
M: passing a `Date(timeInterval)` make tests deterministic

Reply:
Ups, that is usually my comment 😁

Reply:
learned from the best :D